admin-base/core/src/main/java/com/peregrine/admin/servlets/NodesServlet.java

```diff
@@ -232,6 +237,7 @@ private void convertResource(JsonResponse json, ResourceResolver rs, String[] se
                     }
 
                     json.writeAttribute(ANY_DESCENDANT_ACTIVATED, ReplicationUtil.isAnyDescendantReplicated(child));
+                    json.writeAttribute("isReferenced", referenceLister.isReferenced(child));
```

Comment:
not sure if you want to turn this into a constant

Reply:
Done.